index.html

```diff
@@ -1,155 +1,47 @@
-        </div>
-        <br>
-        <div>
-            <label for="range">Your programming level</label><br>
-            <input id="range" type="range" name="level" min="0" max="10">
-        </div>
-        <div>
-            <p>What does HTML stand for?</p>
-            <input id="1" type="checkbox" name="answer_html" value="false">
-            <label for="1">Hyperlinks and text Markup Language</label><br>
-            <input id="2" type="checkbox" name="answer_html" value="false">
-            <label for="2">Home Tool Markup Language</label><br>
-            <input id="3" type="checkbox" name="answer_html" value="true" checked>
-            <label for="3">Hyper Text Markup Language</label>
-        </div>
-        <div>
-            <p>What does HTTP stand for?</p>
-            <input id="radio_1" type="radio" name="answer_http" value="false">
-            <label for="radio_1">Hyper Text Translate Protocol</label><br>
-            <input id="radio_2" type="radio" name="answer_http" value="false">
-            <label for="radio_2">Hyper Text Transmission Protocol</label><br>
-            <input id="radio_3" type="radio" name="answer_http" value="true" checked>
-            <label for="radio_3">Hyper Text Transfer Protocol</label>
-        </div>
-        <br>
-        <div>
-            <label for="comment">Comment:</label><br>
-            <textarea id="comment" name="comment" cols="30" rows="5" maxlength="500">Value</textarea>
-        </div>
-        <br>
-        <div>
-            <label for="select">Which attributes does select tag have?</label><br><br>
-            <select id="select" name="select_attr">
-                <option value="" selected>Your answer</option>
-                <option value="option_true">option</option>
-                <option value="color">color</option>
-                <option value="multiple">display</option>
-            </select>
-        </div>
-        <br>
-        <input type="submit" value="Send">
-    </fieldset>
-</form>
-
+<div class="offer-container">
+    <h3 class="offer-title">What do we offer</h3>
+    <div class="offer-title-line"></div>
+    <div class="offer-blocks-wrapper">
+        <div class="block">
+            <div class="img-container">
+                <img src="./img/project_images/support.png" alt="image:support">
+            </div>
+            <p class="block-info">Support 24/7</p>
+        </div>
+        <div class="block">
+            <div class="img-container">
+                <img src="./img/project_images/communicate.png" alt="image:communicate">
+            </div>
+            <p class="block-info">Communicate directly</p>
+        </div>
+        <div class="block">
+            <div class="img-container">
+                <img src="./img/project_images/book.png" alt="image:book-online">
+            </div>
+            <p class="block-info">Book online</p>
+        </div>
+        <div class="block">
+           <div class="img-container">
```

Comment:
**Just recommendation, it's extra info for now**
Don't add wrappers if you can escape them
for example you could use only `<img> `here
```
img {
    width: 100px;
    height: 100px;
   object-fit: contain;
}
```
[Read more](https://www.w3schools.com/css/css3_object-fit.asp)

Reply:
Removed extra wrapper and added object-fit: contain.